src/components/search/searchHeader/searchHeader.module.scss

```diff
@@ -0,0 +1,100 @@
+@use "breakpoints";
+@import "variables";
+
+.searchHeader {
+  --background-color: #{$color-engel-medium-light};
+  grid-column: 1 / -1 !important;
+
+  // Give increased z-index to show mobile actions
+  z-index: 2;
+
+  // Menu that contains the search form
+  .searchMenu {
+    padding: $spacing-xs;
+    // Fullscreen minus the main navigation height
+    height: calc(100vh - 50px);
+    background-color: var(--background-color);
+    overflow-y: hidden;
+
+    @include breakpoints.respond-above(m) {
+      height: auto;
+      display: grid;
+      grid-template-rows: 1fr;
+      grid-template-columns: 4fr 1fr;
+      grid-column-gap: $spacing-s;
+
+      .closeSearch {
+        margin-left: auto;
+        width: 230px;
+        height: 56px;
+      }
+    }
+  }
+
+  .searchActions {
+    background-color: var(--background-color);
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    position: fixed;
+    bottom: 0;
+    left: 0;
+    right: 0;
+
+    @include breakpoints.respond-above(m) {
+      position: initial;
+      padding: $spacing-xs $spacing-s;
+    }
+
+    .countAndTags {
+      width: 0;
+      visibility: hidden;
+
+      @include breakpoints.respond-above(m) {
+        visibility: visible;
+        width: auto;
+        margin-left: $spacing-layout-l;
+      }
+    }
+
+    button {
+      width: 50%;
+
+      @include breakpoints.respond-above(m) {
+        width: 200px;
+        border: 2px solid;
+        height: 45px;
```

Comment:
Why `200px` and `45px`?

In essence we are customising HDS so it'd be good to highlight  why so that we don't accidentally discard the customisations.

If you have set a fixed width because you are trying to undo the `width: 50%;`, you could for instance do `width: unset;`.

In this view you are transforming a supplementary HDS button into a secondary HDS button with custom styles. One thing to note is that with supplementary buttons, the button label has no left margin whereas with secondary buttons they do.

<img width="507" alt="Screenshot 2021-07-08 at 11 03 10" src="https://user-images.githubusercontent.com/9090689/124885605-1dc23c80-dfdc-11eb-88c7-97a8ff08bf08.png">
<img width="507" alt="Screenshot 2021-07-08 at 11 03 05" src="https://user-images.githubusercontent.com/9090689/124885617-1f8c0000-dfdc-11eb-9cd5-b9b141fbbee4.png">

Reply:
I was indeed trying to undo the `width: 50%`, unset seems to do the trick. Height is set to `45px` because according to design the buttons should be "slimmer". I didn't even notice that the secondary button had left margin, this needs a little tweaking.